Mathlib/Data/Sum/Basic.lean

```diff
@@ -201,6 +202,75 @@ section LiftRel
 #align sum.lift_rel.swap Sum.LiftRel.swap
 #align sum.lift_rel_swap_iff Sum.liftRel_swap_iff
 
+mk_iff_of_inductive_prop Sum.LiftRel Sum.liftRel_iff
+
+variable {r : α → γ → Prop} {s : β → δ → Prop} {x : Sum α β} {y : Sum γ δ}
+  {a : α} {b : β} {c : γ} {d : δ}
+
+theorem isLeft_eq_of_liftRel (h : LiftRel r s x y) : x.isLeft = y.isLeft := by
```

Comment:
```suggestion
theorem LiftRel.isLeft_congr (h : LiftRel r s x y) : x.isLeft = y.isLeft := by
```
but also ugh that would be nicer as
```suggestion
theorem LiftRel.isLeft_congr (h : LiftRel r s x y) : x.IsLeft ↔ y.IsLeft := by
```

Reply:
Happy to change that - in the time since I made this and now, Lean no longer seems to expand bools to "Bool = true" in the same way.

Reply:
Hmm, it does actually. Well, if you think it's nicer I will change it. The thing is that sometimes you want to transform the expression itself... it's essentially down to which side of `Bool.eq_iff_iff` you prefer. (For what it's worth, changing to iff breaks some of my proofs in the same file.)

Reply:
Nono, I meant that I would like  a Prop-valued `IsLeft` instead of the Bool-valued `isLeft`, but that's out of scope for this PR. Just pick the first suggestion :)

Reply:
The iff is better anyway, changed.